### PR TITLE
install inode plugin as systemd or upstart services

### PR DESCRIPTION
**Pull request by jbabe**
- install requirements via pip
- Update readme

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+  become: true
+  user: ubuntu
+  vars_files:
+    - vault.yml
+    - vars.yml
+  tasks:
+   - name: Install inode monitor on server
+     git:
+       repo: https://github.com/Foild/newrelic-inode-plugin.git
+       dest: /usr/local/bin/newrelic-inode-plugin
```
Comment (rubene) [inline tasks/main.yml]: this `/usr/local/bin/newrelic-inode-plugin` is getting use on different places can be a role deafult?

File: tasks/main.yml [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+       force: yes
+
+   - command: bundle install
+     args:
+       chdir: /usr/local/bin/newrelic-inode-plugin
+
+   - file:
```
Comment (rubene) [inline tasks/main.yml]: some of the task have names some no, can you add a name to describe what each task is doing?

**Review by rubene — COMMENTED** [review]
💩

File: tasks/main.yml (hunk 1 above)
Comment (jbabe) [reply tasks/main.yml]: yep...added a var now.

File: tasks/main.yml (hunk 2 above)
Comment (jbabe) [reply tasks/main.yml]: yeah - added now (no names breaks `ansible-lint` too)

**Comment by jbabe** [comment]
👋

**Comment by rubene** [comment]
🐘 🐘 🐘

**Pull request merged**